api/proto/config/devmodel.proto

```diff
@@ -34,16 +57,23 @@ message sWAdapterParams {
 	repeated string bondgroup = 10;
 }
 
+// systemAdapters, are the higher l2 concept built on physicalIOs.
+// systemAdapters, gives all the required bits to turn the physical IOs
+// into useful IP endpoints
+// These endpoints can be further used to connect to controller or 
+// can be shared between workload/services running on the node.
 message SystemAdapter {
 	// name of the adapter; hardware-specific e.g., eth0
 	string name = 1;
 
 	sWAdapterParams allocDetails = 20;
 
 	// this is part of the freelink group
+	// deprecate: look at PhysicalIO->Usage
```

Comment:
As I said in the other review just now, we don't have a replacement for this free bit.

I think it is different than the Usage we have defined, which is why a suggested a different message to carry a Policy or UsagePolicy if you'd like. For now that is just "free", but later could be rate limits, monthly max, who knows.

Reply:
okay. That makes sense, its additional color on primary usage field. I will make this change.

Reply:
Done added usage policy